Twenty403/src/main/java/org/firstinspires/ftc/twenty403/command/autonomous/left/AutoLeftCycleLeft.java

```diff
@@ -18,25 +14,25 @@ public AutoLeftCycleLeft(Robot r) {
                 AutoConstants.Left.START_TO_E_JUNCTION
             )
                 .alongWith(
-                    new LiftHighJunctionCommand(r.liftSubsystem),
-                    new ClawOpenCommand(r.clawSubsystem),
+                    r.liftSubsystem.highCommand,
```

Comment:
I'm not sure if I like this change or not. On one hand, no need to use new objects. But on the other hand, The hierarchy is broken, I think. A highCommand belongs to the liftSubsystem doesn't roll on the tongue as nicely as a highCommand uses the liftSubsystem.

Reply:
This is the primary change that I'm not strictly happy with. I sort of like the idea of making "single subsystem" commands nested classes, but I was able to eliminate most of them by enabling the methodRef stuff. The only place the commands come up now are in autonomous command groups. This is probably worth chatting with the students about (Particularly our veteran programmers, as both of them were involved with some of this stuff last year)